lib/travis/build/script/scala.rb

```diff
@@ -6,8 +6,8 @@ class Script
       class Scala < Jvm
 
         DEFAULTS = {
-          scala: '2.10.4',
-          jdk:   'default'
+          scala: '2.12.1',
+          jdk:   'oraclejdk8'
```

Comment:
I'm not sure I understand what exactly you want me to remove - do you mean you want the JDK version to remain on `default`? Scala 2.12 actually requires at least JDK 8 (http://scala-lang.org/news/2.12-roadmap).